src/controller/students.controller.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable prefer-const */
+/* eslint-disable import/no-unresolved */
```

Comment:
remove rules from all of the files except eslintrc

Reply:
done